queue-non-urgent-requests.md

```diff
@@ -0,0 +1,43 @@
+---
+version: 1.0
+submitted_by: greenhsu123
+published_date: TBD
+category: cloud
+tags: 
+ - role:cloud-engineer
+ - size:small
+---
+
+# Queuing non-urgent processing requests
+
+## Description
+
+All systems have periods of peak and low load. From a hardware-efficiency perspective, we are more efficient with hardware if we minimise the impact of requests spikes with an implementation that allows an even utilisation of components. From an energy-efficiency perspective, we are more efficient with energy if we ensure that idle resources are kept to a minimum. 
+
+## Solution
+
+Utilise a message queue, so requests that do not require immediate processing are queued. 
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Sofware Carbon Intensity Spec](https://github.com/Green-Software-Foundation/software_carbon_intensity)
+
+Concerning the SCI equation, optimising peak CPU utilisation will impact two parts:
+
+- `E`: By reducing the total number of computing equipment required, we reduce the total embodied carbon.
```

Comment:
These descriptions should be flipped, E --> Energy, M --> eMbodied emissions

Reply:
Here again, thanks!